Sources/SwiftGodot/Core/ClassServices.swift

```diff
@@ -133,13 +133,14 @@ public class ClassInfo<T:Object> {
         if let returnValue {
             retInfo = returnValue.makeNativeStruct()
         }
-        let functionInfo = Unmanaged.passRetained(FunctionInfo (function, retType: returnValue?.propertyType))
+        let userdata = UnsafeMutablePointer<FunctionInfo>.allocate(capacity: 1)
+        userdata.initialize(to: .init(function, retType: returnValue?.propertyType))
         
```

Comment:
I am a little nervous that we used to pass a retained value, and now we do not.

That said, it looks like we never released the value in the past.   Maybe it was always persisted.

Reply:
Yeah, it's a registration thing and always persisting. Using `Unmanaged` is an overkill here, imo. Even if we bring deregistration logic, it would be enough just to deinitialize and deallocate the `FunctionInfo`